timemachine/fe/rbfe.py

```diff
@@ -250,10 +262,11 @@ def setup_optimized_initial_state(
     temperature: float,
     seed: int,
 ) -> InitialState:
-    # Use pre-optimized initial state with the closest value of lambda as a starting point for optimization.
+    """Use pre-optimized initial state with the closest value of lambda as a starting point for optimization.
 
-    # NOTE: The current approach for generating optimized conformations in `optimize_coordinates` creates a
-    # discontinuity at lambda=0.5. Ensure that we pick a pre-optimized state on the same side of 0.5 as `lamb`:
+    NOTE: The current approach for generating optimized conformations in `optimize_coordinates` creates a
```

Comment:
nit: this isn't really intended to be part of the docstring. It could be adapted to go in the docstring, but e.g. should probably get rid of the trailing `:`.

Reply:
Good catch, moved back to being an inline comment in https://github.com/proteneer/timemachine/pull/1336/commits/197fcaa79da1e3a2112fbc5aa8f5d8b69f2d73ef and removed the trailing `:`